src/share/cxx/vector/KokkosKernels_Vector_SIMD.hpp

```diff
@@ -42,6 +42,25 @@
 #ifndef __KOKKOSKERNELS_VECTOR_SIMD_HPP__
 #define __KOKKOSKERNELS_VECTOR_SIMD_HPP__
 
+#if defined(__INTEL_COMPILER)
+
+#define NO_VECTOR_DEPS _Pragma("simd")
+#define ALWAYS_VECTORIZE _Pragma("vector always")
+
+#elif defined(__GNUG__)
+#if(__GNUG__ == 4 && __GNUC_MINOR__ >= 9) || __GNUG__ > 4
+
+#define NO_VECTOR_DEPS _Pragma("GCC simd")
```

Comment:
How about putting this whole thing in a HommexxMacros.hpp file (or whatever)? Can use this in ExecSpaceDefs.hpp (also a fine place for it) and a couple of other spots (e.g., ESF).